src/controllers/DynamicCategoryFacetQueryController.ts

```diff
@@ -55,7 +56,29 @@ export class DynamicCategoryFacetQueryController {
     };
   }
 
-  public get currentValues(): IFacetRequestValue[] {
+  public executeIsolatedQuery(): Promise<IQueryResults> {
```

Comment:
It's an isolatedQuery, that doesn't affect the rest or the page, and isn't counted as a full fledged query API side because the numberOfResults is of 0.
It's used for the show more/less values

Reply:
Ok, I still find the current name a little vague though. Would a name like `getFacetValues` or `getFacetValuesForLastQuery` be clearer? The word `execute` especially suggests a `void` method, but we are returning a promise resolving with query results.